skrub/dataframe/tests/test_namespace.py

```diff
@@ -0,0 +1,37 @@
+import pandas as pd
+import pytest
+
+import skrub.dataframe._pandas as skrub_pd
+import skrub.dataframe._polars as skrub_pl
+from skrub.dataframe import POLARS_SETUP, get_df_namespace
+
+main = pd.DataFrame(
+    {
+        "userId": [1, 1, 1, 2, 2, 2],
+        "movieId": [1, 3, 6, 318, 6, 1704],
+        "rating": [4.0, 4.0, 4.0, 3.0, 2.0, 4.0],
+        "genre": ["drama", "drama", "comedy", "sf", "comedy", "sf"],
+    }
+)
+
+
+def test_get_namespace_pandas():
+    skrub_px, _ = get_df_namespace(main, main)
+    assert skrub_px is skrub_pd
+
+    with pytest.raises(TypeError, match=r"(?=.*Pandas or Polars)(?=.*supported)"):
```

Comment:
Can this be more specific? Because there are two similar errors that can be identified with this (`Only ... supported` or `Mixing ... not supported`), and I think we want to catch the `Only` case here?